log.go

```diff
@@ -412,6 +414,7 @@ func (l Logger) Write(p []byte) (n int, err error) {
 		// Trim CR added by stdlog.
 		p = p[0 : n-1]
 	}
+	e.CallerSkipFrame(1)
```

Comment:
`e` is not declared, does this compile?

Reply:
It does not compile.  I blame my brain, as unhelpful as that is. :)

Reply:
Fixed. 99c9830747e5fbe673073f2600e78ba9758cf0df